### PR TITLE
Added Docs for Overview & workflow Analytics.

### PR DESCRIPTION
**Pull request by Jonsy13**
Signed-off-by: Jonsy13 <vedant.shrotria@chaosnative.com>

**What this PR does / why we need it**:

**This PR will add docs for Overview & workflow analytics.**

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Checklist:**
-   [ ] Fixes #<issue number>
-   [X] Signed the commit for DCO to be passed
-   [X] Labelled this PR & related issue with `documentation` tag

### CODE REVIEW

File: website/docs/workflow-analytics.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+sidebar_label: Workflow Analytics
+---
+
+---
+
+Once users have successfully scheduled a workflow (Recurring/ Non-recurring), The users can check how there workflows are running, analyse the workflow runs Hourly/daily/Monthly using workflow dashboards and check how their system behaves when chaos injection happens using Application dashboards.
+
+### Overview
+
+Overview Tab helps users to manage their connected Datasources and Dashboards & makes it easier to view analyse the workflows. Overview Tab is divided into 3 parts -
```
Comment (Saranya-jena) [inline website/docs/workflow-analytics.md]: `Datasources` can be changed to `Data sources`

File: website/docs/workflow-analytics.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
```
Comment (Saranya-jena) [inline website/docs/workflow-analytics.md:14]: `easier to view analyse ` -> `easier to view and analyze`

File: website/docs/workflow-analytics.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+- Data Sources
+- Dashboards
+- Workflow Dashboards
+
+> When no agent is connected or in active state, the overwiew page will show a banner for connecting an agent. Users can deploy a new agent using [litmusctl](https://github.com/litmuschaos/litmusctl).
+
+<img src={require('./assets/overview-agent-connect.png').default} width="1000" height="500" />
+
+Once the agent is connected, Users will see a banner for adding datasources and list of recently updated workflow runs (if any).
```
Comment (Saranya-jena) [inline website/docs/workflow-analytics.md]: - `Users` -> `users`
- `datasources` can be changed to `data sources`

File: website/docs/workflow-analytics.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+<img src={require('./assets/overview-first-look.png').default} width="1000" height="500" />
+
+For Adding a Data source, Users can follow here
```
Comment (Saranya-jena) [inline website/docs/workflow-analytics.md]: - `Users` -> `users`
- `here` => `here:`

File: website/docs/workflow-analytics.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+Once users have added a data source sucessfully, they will be able to see the data source listed on the overview page.
+
+<img src={require('./assets/overview-with-datasource.png').default} width="1000" />
+
+Now, users will get an banner with an option to add a dashboard for visualizing the data from the connected the data sources.
```
Comment (Saranya-jena) [inline website/docs/workflow-analytics.md]: - `get an` -> `get a`
- remove `the` after `connected`

File: website/docs/workflow-analytics.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+
+For Adding a Dashboard, Users can follow here, After setting up the dashboard, the users will be able to see the the dashboard on overview page.
```
Comment (Saranya-jena) [inline website/docs/workflow-analytics.md]: - `Users` -> `users`
- `After` -> `after`
- remove a duplicate `the`

File: website/docs/workflow-analytics.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+
+<img src={require('./assets/overview-with-dashboard.png').default} width="1000" />
+
+Users can check workflow analytics and workflow run graph by clicking on _**see workflow run**_ & _**see workflow analytics**_ icons for a workflow from list of workflows shown on overview page.
+
+### Workflow Analytics
+
+As we have already discussed about the workflow run graph in other parts of documentation, we can proceed with workflow analytics. On clicking _**see workflow analytics**_, Users will be redirected to workflow analytics page for that particular workflow.
```
Comment (Saranya-jena) [inline website/docs/workflow-analytics.md]: - Users -> users

File: website/docs/workflow-analytics.md [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+
+<img src={require('./assets/workflow-analytics.png').default} width="1000" height="500" />
+
+On workflow analytics page, In the first container, user will be able to see the details about the selected workflow like Worklflow name, workflow id, agent details like name and namespace and workflow run details like last run and it's Regularity.
```
Comment (Saranya-jena) [inline website/docs/workflow-analytics.md]: - `In` -> `in`
- `name and namespace and workflow run details` -> `name, namespace, and workflow run details`
- `it's` -> `its`

File: website/docs/workflow-analytics.md [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+
+On clicking on **show statistics**, we can see the experiment statistics, average resiliency score and pass/fail ratio for the workflows/Experiments.
+
+<img src={require('./assets/workflow-single-run.png').default} width="1000"/>
+
+If selected workflow is a recurring workflow or the selected is ran for multiple times, then it will show Average Resiliency Score for all workflow runs, and also users will be able to switch between workflows & experiments using dropdown for checking Pass/Fail ratio.
```
Comment (Saranya-jena) [inline website/docs/workflow-analytics.md]: - `selected is ran for` -> `selected one is run for`

File: website/docs/workflow-analytics.md [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+
+<img src={require('./assets/workflow-multi-run.png').default} width="1000"/>
+
+If the Selected workflow is a recurring workflow users will get a heatmap, showing all the workflow runs for the selected workflow. On hovering over the points in heatmap, we can see the average resiliency score for that particular day.
```
Comment (Saranya-jena) [inline website/docs/workflow-analytics.md]: - `Selected` -> `selected`
- `recurring workflow` -> `recurring workflow,`

File: website/docs/workflow-analytics.md [hunk 11]
```diff
@@ -0,0 +1,67 @@
...
+
+You can check the workflow analytics and the workflow run graph by clicking on _**see workflow run**_ & _**see workflow analytics**_ icons for a workflow from the list of workflows present on the overview page.
+
+### Workflow Analytics
+
+As we have already discussed the workflow run graph in other parts of documentation, this section will focus on workflow analytics. On clicking _**see workflow analytics**_, you will be redirected to the workflow analytics page for that particular workflow.
+
+<img src={require('./assets/workflow-analytics.png').default} width="1000" height="500" />
+
+On the workflow analytics page, In the first container, you will be able to see the details about the selected workflow such as the workflow name, workflow id, agent details(name, namespace, etc.), and the workflow run details like the last run as well as regularity.
```
Comment (Saranya-jena) [inline website/docs/workflow-analytics.md]: `In` -> `in`

File: website/docs/workflow-analytics.md [hunk 12]
```diff
@@ -0,0 +1,67 @@
...
+
+If the selected workflow is recurring or is re-run multiple times, a heat map representing the days of the year showing all the workflow runs for the selected workflow can be observed. On hovering over a point in the heat map, the average resiliency score for that particular day will be shown in the form of a tool-tip.
+
+<img src={require('./assets/workflows-heatmap.png').default} width="1000"/>
+
+On clicking on a heat map entry, a stacked bar graph representing the pass/fail ratio is shown, on which you can see the workflow runs on that day and hover over the bars to see the resiliency for that particular workflow run on that day.
+
+<img src={require('./assets/workflow-activity.png').default} width="1000"/>
+
+On clicking on a bar, more details about the workflow run represented by that stacked bar are presented in a tabular format. Here, we can see a table showing test names, experiment names, test results & weights, etc.
```
Comment (Saranya-jena) [inline website/docs/workflow-analytics.md]: `we` -> `you`

**Review by Saranya-jena — APPROVED** [review]
LGTM, thank you for your efforts 🚀

**Review by vanshBhatia-A4k9 — APPROVED** [review]

**Review by SarthakJain26 — APPROVED** [review]

**Review by arkajyotiMukherjee — APPROVED** [review]

**Pull request merged**